contracts/limitOrder/LimitOrderBook.sol

```diff
@@ -219,4 +228,63 @@ contract LimitOrderBook is
 
         return signer;
     }
+
+    function _verifyTriggerPrice(LimitOrder memory order, uint80 roundIdWhenTriggered) internal view {
+        if (order.orderType == ILimitOrderBook.OrderType.LimitOrder) {
+            return;
+        }
+
+        // NOTE: Chainlink proxy's roundId is always increased
+        // https://docs.chain.link/docs/historical-price-data/
+
+        // LOB_IRI: Invalid RoundId
+        require(order.roundIdWhenCreated > 0 && roundIdWhenTriggered > order.roundIdWhenCreated, "LOB_IRI");
```

Comment:
Is it possible that `roundIdWhenTriggered == order.roundIdWhenCreated`?

Reply:
Ah yes, I think it's possible. Will change to `roundIdWhenTriggered >= order.roundIdWhenCreated`.